tests/impl.cpp

```diff
@@ -6081,7 +6081,16 @@ result_t test_mm_stream_si32(const SSE2NEONTestImpl &impl, uint32_t iter)
 
 result_t test_mm_stream_si64(const SSE2NEONTestImpl &impl, uint32_t iter)
 {
-    return TEST_UNIMPL;
+    const int64_t a = (const int64_t) impl.mTestInts[iter];
+#if defined(__x86_64__) || defined(__i386__) || \
+    (defined(_WIN32) && defined(__LLP64__))
```

Comment:
Why `_WIN32` **AND** `__LLP64__`?

Reply:
Or should I remove the `(defined(_WIN32) && defined(__LLP64__))` case?

If I conduct the following way, then it won't meet the implementation on Linux on x86 machine

```c
#if (defined(__x86_64__) || defined(__i386__)) && (defined(_WIN32) || defined(__LLP64__))
    long long p[1];
#else
    int64_t p[1];
#endif
```

Reply:
Any progress?